src/services/codefixes/convertTypedefToType.ts

```diff
@@ -48,35 +50,63 @@ registerCodeFix({
             ];
         }
     },
-    getAllCodeActions: context => codeFixAll(context, errorCodes, (changes, diag) => {
-        const node = getTokenAtPosition(diag.file, diag.start);
-        if (node) doChange(changes, node, diag.file);
-    })
+    getAllCodeActions: context => codeFixAll(
+        context,
+        errorCodes,
+        (changes, diag) => {
+            const node = getTokenAtPosition(diag.file, diag.start);
+            if (node) doChange(changes, node, diag.file, context);
+        }
+    )
 });
 
-function doChange(changes: textChanges.ChangeTracker, node: Node, sourceFile: SourceFile) {
+function doChange(
+    changes: textChanges.ChangeTracker,
+    node: Node, sourceFile: SourceFile,

```

Comment:
If you multi-line these, can you put each parameter on its own line instead of mixed counts?